cli.js

```diff
@@ -27,14 +27,18 @@ try {
             includePackages: commander.packages,
         }).forEach(logLine => console.log(logLine))
     }else {
-        const dedupedYarnLock = fixDuplicates(yarnLock,{
+        let dedupedYarnLock = fixDuplicates(yarnLock, {
             useMostCommon: commander.mostCommon,
             includePackages: commander.packages,
         });
 
         if (commander.print) {
             console.log(dedupedYarnLock);
         } else {
+            const eolMatch = yarnLock.match(/(\r?\n)/);
```

Comment:
I was about to suggest that we pass the right sequence for EOL to `fixDuplicates` as an option and use it when concatenating the results, instead of always using `\n` and then maybe replacing it with `\r\n`. But turns out that the module doing the contactenation is `@yarn\lockfile` that doesn't support anything other than `\n` 🤦‍♂️ 

I'm ok with leaving this logic in the CLI then.

Reply:
Yeah I did look there first. It's a bit unfortunate ☹️